project/MiMa.scala

```diff
@@ -715,6 +715,14 @@ object MiMa extends AutoPlugin {
 
         // #19390 Add flow monitor
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOpsMat.monitor")
+      ),
+      "2.4.4" -> Seq(
```

Comment:
Not sure if I needed to add a new version to the map or just add the filters to 2.4.2